rust/agama-server/src/web/docs.rs

```diff
@@ -49,6 +49,20 @@ use utoipa::OpenApi;
         crate::storage::web::iscsi::nodes,
         crate::storage::web::iscsi::update_initiator,
         crate::storage::web::iscsi::update_node,
+        crate::storage::web::dasd::probe,
+        crate::storage::web::dasd::supported,
+        crate::storage::web::dasd::devices,
+        crate::storage::web::dasd::format,
+        crate::storage::web::dasd::enable,
+        crate::storage::web::dasd::disable,
+        crate::storage::web::dasd::set_diag,
```

Comment:
as we obviously format about dasd in previous PR maybe it is time to consider something like https://github.com/rxdiscovery/utoipa_auto_discovery ?